src/oaklib/cli.py

```diff
@@ -2271,6 +2273,40 @@ def lexmatch(output, recreate, rules_file, lexical_index_file, add_labels):
     Using custom rules:
 
         runoak  -i foo.obo lexmatch -R match_rules.yaml -L foo.index.yaml -o foo.sssom.tsv
+
+    Example of a LexicalIndex object for the term 'continuant':
+
+    LexicalIndex
```

Comment:
I think this is way too much for the CLI docs. This isn't even a real use case for doing things with this, any more than there is for introspecting a blast index. Most of the time the user doesn't need to know this exists. On some occasions for large ontologies it's use to persist this to do repeated runs without re-indexing, but you don't even need to look at it.

The right place to put this is in the **datamodel documentation**. All objects in OAK conform to a datamodel. This one is documented here: https://incatools.github.io/ontology-access-kit/datamodels/lexical-index/index.html

The CLI docs should just point to this. Adding examples to this would be great, but do it in the yaml!

cc @matentzn